pkg/Microsoft.NETCore.Platforms/runtime.json

```diff
@@ -7,6 +7,26 @@
             "#import": [ "base" ]
         },
 
+        "android": {
+            "#import": [ "any" ]
+        },
+        "android-arm": {
+	    "#import": [ "any" ]
+        },
+        "android-arm64": {
+	    "#import": [ "any" ]
```

Comment:
I think the spacing is still off. You should indent the "#import" node, like on line 11. I think the previous feedback was about you using tabs instead of spaces.